posthog/models/feedback/survey.py

```diff
@@ -277,7 +282,7 @@ def update_survey_iterations(sender, instance, *args, **kwargs):
     instance.iteration_start_dates = list(
         rrule(
             DAILY,
-            count=iteration_count,
+            count=max(iteration_count, MAX_ITERATION_COUNT),
```

Comment:
It's still wrong, tho 🤣. I might be seeing an outdated version tho

Reply:
good job I've not been using git for more than a decade 🙈

Reply:
git is hard, let's go back to passing floppy disks around the office

Reply:
(Not that I've ever done that, I was a baby)

Reply:
🤣 

in my last job I found a team where only one of them built the software, and they put it on a CD for someone else to install

Reply:
This makes me think you were at that job in 1995

Reply:
sadly no